packages/msw-addon/package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Mock API requests in Storybook with Mock Service Worker.",
   "version": "1.4.1",
   "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
```

Comment:
Without this, there will be no connection between the source code and the emitted type declarations.

Reply:
Interesting! I didn't set it because it's an index.d.ts so that would by default be found by typescript. I did try the code out in the example project. I think it's better to set it like you did though, it's more explicit.

I believe the typing lookup functionality from typescript does something like:
- Find typings in "typings" property from package.json, if not
- Find declared modules in <root>/index.d.ts, if not
- Find node_modules/the-lib/dist/index.d.ts, if not
- Find node_modules/@types/the-lib/dist/index.d.ts, if not give up